times_reader/__main__.py

```diff
@@ -304,9 +304,10 @@ def convert_parameter(tablename: str, df: DataFrame):
 
     sets = {m.times_name for m in config.times_xl_maps if "VALUE" not in m.col_map}
 
-    # Compute map fname -> tables: right now ALL_TS -> ts.dd, rest -> output.dd
+    # Compute map fname -> tables: put ALL_TS and MILESTONYR in separate files
     tables_in_file = {
         "ts.dd": ["ALL_TS"],
+        "milestonyr.dd": ["MILESTONYR"],
```

Comment:
I decided to call it `milestony.dd` instead of `milestonyr.inc` so that our existing `dd_to_csv.py` script will automatically convert it and count this table as existing in the ground truth, so once we add this file to `demos-dd`, the additional rows regression should be resolved

Reply:
Actually this is what we would like avoid, because in the existing workflow milestone years are not generated into a dd file, but are added to the run file. Also we should be able to pass the name of the milestone year definition for to the tool to use.

Reply:
I don't understand. We want to use the milestone years as computed by our tool from the period definitions in the input XL files, right? Then the tool must output it somewhere, whether the run file or a separate file. (and it's cleaner for it to be output to a separate file than for us to find the appropriate line in `scenario.run` and modify it)

And similarly, we need to know the ground truth values of milestone years for the `demos-dd`. Otherwise how do we check if our tool is computing them correctly?

Perhaps I'm missing something obvious :)

Reply:
Sorry for not being clear on this one. We would need to expand our testing to do this correctly. If we want to test whether our tool generates correct milestone years, we should include information on all the milestone years definitions in the benchmarks and then compare those to the ones generated by the tool.

Reply:
> we need to be given the ground truth values of milestone years for the demos-dd

**Comment**: There is no unique ground truth to that, other than what I have said before:  The actual set used in any model run must be a **subset** of the full set of milestone years according to the active **period definition**, such that when the subset contains N members, those members must be equal to the first N years from that full set.  It is a free choice of the user to select the period definition (if several have been defined) and then the subset thereof when launching a model run, and neither choice affects the contents of the **DD files**. Therefore, I think the tool should also support making that choice, like VEDA does.

Reply:
Yes, we plan to add a command line flag to enable the user to pick. But for the tests it is sufficient for the ground truth to contain any valid milestone years.